### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by ik-web**

### CODE REVIEW

File: src/calculateRentalCost.js [hunk 1]
```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const pricePerDay = 40;
+  const fullPrise = days * pricePerDay;
```
Comment (illiarerun) [inline src/calculateRentalCost.js:32]:
```suggestion
  const fullPrice = days * pricePerDay;
```

File: src/calculateRentalCost.js [hunk 2]
```diff
@@ -28,7 +28,21 @@
...
+  const cashbackForTreeDays = 20;
+  const cashbackForSevenDays = 50;
```
Comment (illiarerun) [inline src/calculateRentalCost.js:34]: [Checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md) 5th
item, don't hardcode any values in variable names, value might change, use smth like `basicDiscount`, `advancedDiscount`

File: src/calculateRentalCost.js [hunk 3]
```diff
@@ -28,7 +28,21 @@
...
+  let total = 0;
+
+  if (days >= 7) {
+    total = fullPrise - cashbackForSevenDays;
+  } else if (days >= 3) {
+    total = fullPrise - cashbackForTreeDays;
+  } else {
+    total = fullPrise;
+  }
```
Comment (illiarerun) [inline src/calculateRentalCost.js:43]: [Checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md) 2nd and 3rd items, do you really need this `let` variable? You can return value instantly form `if` block

**Review by illiarerun — CHANGES_REQUESTED** [review]
Nice start, now let's work on some fixes:

File: src/calculateRentalCost.js (hunk 2 above)
Comment (ik-web) [reply src/calculateRentalCost.js]: Fixed

File: src/calculateRentalCost.js (hunk 1 above)
Comment (ik-web) [reply src/calculateRentalCost.js]: Fixed

File: src/calculateRentalCost.js [hunk 4]
```diff
@@ -28,7 +28,18 @@
...
+  const pricePerDay = 40;
+  const fullPrice = days * pricePerDay;
+  const basicDiscount = 20;
+  const advancedDiscount = 50;
+
+  if (days >= 7) {
+    return fullPrice - advancedDiscount;
+  } else if (days >= 3) {
+    return fullPrice - basicDiscount;
+  }
```
Comment (jstmpelowycz) [inline src/calculateRentalCost.js:40]:
```suggestion
  if (days >= 7) {
    return fullPrice - advancedDiscount;
  }
  
  if (days >= 3) {
    return fullPrice - basicDiscount;
  }
```

Comment (jstmpelowycz) [reply src/calculateRentalCost.js:40]: as far as `return` exits the program right away, you could simplify it

**Review by jstmpelowycz — CHANGES_REQUESTED** [review]
almost done, one thing left to fix

File: src/calculateRentalCost.js (hunk 4 above)
Comment (ik-web) [reply src/calculateRentalCost.js]: Fixed. Thank you for the hint!

**Pull request closed without merging**